docs/proposals/PROPOSAL_E.md

```diff
@@ -159,14 +159,7 @@ If a query results in no referrers found, an empty manifest list MUST be returne
 
 #### Registry Upgrade Expectations
 
-When upgrading existing registries, the following manifests MUST be scanned for a `refers` field:
-
-1. Any existing `application/vnd.oci.image.manifest.v1+json` manifest referenced by a [digest tag](#digest-tags)
-2. Any newly uploaded `application/vnd.oci.image.manifest.v1+json` and `application/vnd.oci.artifact.manifest.v1+json` manifests
-
-Clients SHALL NOT expect manifests uploaded before the [referrers API](#referrers-api) is available on that registry, without using a [digest tag](#digest-tags), will be included in future API responses.
-
-#### Digest Tags
+##### Digest Tags
```

Comment:
The Digest Tags are an alternative to the API, so it feels strange putting it as a child of that section. It would probably make sense to retitle `Registry HTTP API` to `distribution-spec changes` and put the workflow to use one or the other at the top. `Ordering and Pagination` should probably be bumped down one level to be a child of `Referrers API`.